### PR TITLE
Rename `enrollUserInMfaFactor` to `enrollAuthFactor`

### PR DESCRIPTION
**Pull request by jonatascastro12**
## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Comment by jonatascastro12** [comment]
Current dependencies on/for this PR:
* main
  * **PR #853** <a href="https://app.graphite.dev/github/pr/workos/workos-node/853" target="_blank">Graphite</a> 
    * **PR #854** <a href="https://app.graphite.dev/github/pr/workos/workos-node/854" target="_blank">Graphite</a> 
      * **PR #855** <a href="https://app.graphite.dev/github/pr/workos/workos-node/855" target="_blank">Graphite</a> 
        * **PR #856** <a href="https://app.graphite.dev/github/pr/workos/workos-node/856" target="_blank">Graphite</a> 
          * **PR #857** <a href="https://app.graphite.dev/github/pr/workos/workos-node/857" target="_blank">Graphite</a> 
            * **PR #858** <a href="https://app.graphite.dev/github/pr/workos/workos-node/858" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/workos/workos-node/858?utm_source=stack-comment).

File: src/users/serializers/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 export * from './authenticate-user-with-code-options.serializer';
 export * from './authenticate-user-with-magic-auth-options.serializer';
 export * from './authenticate-user-with-password-options.serializer';
+export * from './authenticate-user-with-totp-options.serializer';
```
Comment (lukemorales) [inline src/users/serializers/index.ts:4]: Looks like this change doesn't belong in this PR?

**Review by lukemorales — APPROVED** [review]
